packages/manager/apps/container/src/core/routing/iframe-app-router.tsx

```diff
@@ -119,8 +119,8 @@ export function IFrameAppRouter({
         .map((config) => makeRoute({ ...config, iframeRef })),
     [mockedConfiguration],
   );
-  const redirections = useMemo(() => Redirections(configuration), [
-    configuration,
+  const redirections = useMemo(() => Redirections(mockedConfiguration), [
```

Comment:
If all the FF are migrated shouldn't the mockedConfiguration be removed ?

Reply:
Actually not yet.
I tried without it and it wasn't working.
As far as I understood it, `pci-load-balancer` and `octavia-load-balancer` match the same url.
Thus, while both are present in the configuration, `octavia-load-balancer` will be chosen (because first in the list I guess), which makes it not working properly.
That's why this PR is the first step.
I first have to change the feature name from `octavia-load-balancer` to `pci-load-balancer`.
Then, we can remove `octavia-load-balancer` from feature-availability and bff.
And finally, I can remove the `mockedConfiguration`